chrome-extension/sidebar/sidebar.js

```diff
@@ -48,6 +48,290 @@ const htmlToElement = (html) => {
   return template.content.firstChild;
 };
 
+// Begin code for sorting course list
+
+// dropdown element
```

Comment:
This is a lot of code and having to leave comments like "begin code for x" and "end code for x" are signals that you should extract this code into a different file. 

You should extract the relevant code out and update the manifest.json file to include that new one. Perhaps call this folder "schedule-sorting" or similar.